mvlearn/decomposition/permica.py

```diff
@@ -0,0 +1,117 @@
+from .baseica import BaseICA
+from multiviewica import permica
+
+
+class PermICA(BaseICA):
+    r"""
+    Performs one ICA per view (ex: subject) and align sources
+    using the hungarian algorithm.
+
+    Parameters
+    ----------
+    n_components : int, optional
+        Number of components to extract. If None, n_components is set to
+        the minimum number of features in the dataset.
+
+    max_iter : int, default=1000
+        Maximum number of iterations to perform
+
+    preproc: 'pca' or a ViewTransformer-like instance,
+        default='pca'
+        Preprocessing method to use to reduce data.
+        If "pca", performs PCA separately on each view to reduce dimension
+        of each view.
+        Otherwise the dimension reduction is performed using the transform
+        method of the ViewTransformer-like object. This instance also needs
+        an inverse transform method to recover original data from reduced data.
+
+    multiview_output : bool, optional (default True)
+        If True, the `.transform` method returns one dataset per view.
+        Otherwise, it returns one dataset, of shape (n_samples, n_components)
+
+    random_state : int, RandomState instance or None, default=None
+        Used to perform a random initialization. If int, random_state is
+        the seed used by the random number generator; If RandomState
+        instance, random_state is the random number generator; If
+        None, the random number generator is the RandomState instance
+        used by np.random.
+
+    tol : float, default=1e-3
+        A positive scalar giving the tolerance at which
+        the un-mixing matrices are considered to have converged.
+
+    verbose : bool, default=False
+        Print information
+
+    Attributes
+    ----------
+    preproc : ViewTransformer-like instance
+        The fitted instance used for preprocessing
+    W_list : np array of shape (n_groups, n_components, n_components)
+        The unmixing matrices to apply on preprocessed data
+
```

Comment:
For PermICA and MultiviewICA in the docs that are rendered (https://124-206838300-gh.circle-artifacts.com/0/dev/references/decomposition.html#multiview-ica), the inheritted attributes from BaseICA don't seem to show up. Based on this (https://stackoverflow.com/questions/30344289/how-can-i-show-inherited-members-of-a-class-in-my-sphinx-documentation), I think you need to add a line to decomposition.rst for the classes where you want base-class attributes to show up in docs, but not totally sure